[2024-12-22T16:15:02.592+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-22T16:15:02.765+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_top_best_sellers_books.create_author_dim_table manual__2024-12-22T14:55:10.356955+00:00 [queued]>
[2024-12-22T16:15:02.835+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_top_best_sellers_books.create_author_dim_table manual__2024-12-22T14:55:10.356955+00:00 [queued]>
[2024-12-22T16:15:02.840+0000] {taskinstance.py:2866} INFO - Starting attempt 3 of 4
[2024-12-22T16:15:02.926+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): create_author_dim_table> on 2024-12-22 14:55:10.356955+00:00
[2024-12-22T16:15:02.977+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1029) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-22T16:15:03.011+0000] {standard_task_runner.py:72} INFO - Started process 1040 to run task
[2024-12-22T16:15:03.021+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_top_best_sellers_books', 'create_author_dim_table', 'manual__2024-12-22T14:55:10.356955+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/dag_test.py', '--cfg-path', '/tmp/tmp39vt7a3f']
[2024-12-22T16:15:03.074+0000] {standard_task_runner.py:105} INFO - Job 41: Subtask create_author_dim_table
[2024-12-22T16:15:05.109+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_top_best_sellers_books.create_author_dim_table manual__2024-12-22T14:55:10.356955+00:00 [running]> on host c50092bbd7fa
[2024-12-22T16:15:05.432+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_top_best_sellers_books' AIRFLOW_CTX_TASK_ID='create_author_dim_table' AIRFLOW_CTX_EXECUTION_DATE='2024-12-22T14:55:10.356955+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-22T14:55:10.356955+00:00'
[2024-12-22T16:15:05.437+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-22T16:15:05.514+0000] {sql.py:278} INFO - Executing: CREATE TABLE IF NOT EXISTS author_dim (
    author_id SERIAL PRIMARY KEY,  -- Identifiant unique pour chaque auteur
    author VARCHAR NOT NULL        -- Nom de l'auteur
);
[2024-12-22T16:15:05.545+0000] {base.py:84} INFO - Retrieving connection 'books_connection'
[2024-12-22T16:15:05.578+0000] {base.py:84} INFO - Retrieving connection 'books_connection'
[2024-12-22T16:15:05.596+0000] {sql.py:553} INFO - Running statement: CREATE TABLE IF NOT EXISTS author_dim (
    author_id SERIAL PRIMARY KEY,  -- Identifiant unique pour chaque auteur
    author VARCHAR NOT NULL        -- Nom de l'auteur
);, parameters: None
[2024-12-22T16:15:05.682+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-22T16:15:05.704+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_top_best_sellers_books, task_id=create_author_dim_table, run_id=manual__2024-12-22T14:55:10.356955+00:00, execution_date=20241222T145510, start_date=20241222T161502, end_date=20241222T161505
[2024-12-22T16:15:05.831+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-22T16:15:06.609+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-22T16:15:06.623+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
